src/components/list_group/list_group_item.tsx

```diff
@@ -144,6 +144,11 @@ export type EuiListGroupItemProps = CommonProps &
      * By default the text will be same as the label text.
      */
     toolTipText?: string;
+
+    /**
+     * Props can be passed to nested `EuiToolTip`.
```

Comment:
Brain is slightly more awake and capable of actual copy vs just grammar nits today 😅 
```suggestion
     * Allows customizing the tooltip shown when `showToolTip` is true.
     * Accepts any props that [EuiToolTip](/#/display/tooltip) accepts.
```